Leetcode/LinkedListSum/Program.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+
+namespace LinkedListSum
```

Comment:
@TonySkorik, please, provide the link to LeetCode task

Reply:
Done